pkg/editor/interactive.go

```diff
@@ -86,3 +88,28 @@ func CaptureInputFromEditor(template []byte) ([]byte, error) {
 
 	return updatedBytes, nil
 }
+
+// OpenInteractiveEditor opens users default editor populated with a JSON representation of a structure
+func OpenInteractiveEditor(o interface{}, temp string, funcMap template.FuncMap) ([]byte, error) {
+	t := template.New("Template")
+	if funcMap != nil {
+		t.Funcs(funcMap)
+	}
+	dsJsonTemplate, err := t.Parse(temp)
+	if err != nil {
+		return nil, err
+	}
+
+	buff := bytes.NewBuffer([]byte{})
+	err = dsJsonTemplate.Execute(buff, o)
+	if err != nil {
+		return nil, err
+	}
+
+	return CaptureInputFromEditor(buff.Bytes())
+}
+
+// isLastElementOfSlice is a function which is used in HTML templates to determine the last element in a slice
+func IsLastElementOfSlice(index int, lenght int) bool {
+	return index == lenght-1
+}
```

Comment:
add a newline

Reply:
This comment is already addressed in PR #271
The current PR depends on #271 and this is why the change from both PRs are visible here.
Better approach probably was to wait the first PR to be merged.